tests/integration/jobs/jobs.spec.ts

```diff
@@ -158,6 +158,56 @@ describe('job', function () {
         });
       });
 
+      it('should create job with stage in WAITING status when isWaiting flag is true', async function () {
+        const stageWithAwaitingStatus = {
+          ...createJobRequestWithStagesBody.stages[0],
+          isWaiting: true,
+          type: JobName.DEFAULT,
+          data: {},
+          userMetadata: {},
+        };
+        const jobRequest = {
+          ...createJobRequestWithStagesBody,
+          stages: [stageWithAwaitingStatus],
+        };
+        const response = await requestSender.createJob({
+          requestBody: jobRequest,
+        });
```

Comment:
spacing.
separate between arrange, act, and assertions.

Reply:
you right. done